packages/ui/src/file-preview/pdf-preview.tsx

```diff
@@ -5,18 +5,24 @@ import { lazy, memo, Suspense } from "react";
 import { Highlight } from "./types";
 import type { PdfPreviewImplProps } from "./pdf-preview-impl";
 
+const PdfPreviewLazy = lazy<React.ComponentType<PdfPreviewImplProps>>(() =>
```

Comment:
note: this move fixes the reloading issue on PdfPreview reported by simon, see https://llama-index.slack.com/archives/C08HVR9JY87/p1762240087528509